source/reference/compatibility.txt

```diff
@@ -128,6 +128,38 @@ Write Preference Behavior
 legacy :binary:`mongo` shell. To disable retryable writes, use 
 :option:`--retryWrites=false <mongosh --retryWrites>`.
 
+
+ObjectId Methods and Attributes
+-------------------------------
+
+These :ref:`ObjectId() <server-objectid>` methods work differently in
+``mongosh`` than in the legacy ``mongo`` shell.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Method or Attribute
+     - ``mongo`` Behavior
+     - ``mongosh`` Behavior
+
+   * - ``ObjectId.str``
+     - | Returns a hexadecimal string:
+       | ``6419ccfce40afaf9317567b7``
+     - | Undefined
+       | (Not available)
+
+   * - ``ObjectId.valueOf()``
+     - | Returns the value of ``ObjectId.str``:
+       | ``6419ccfce40afaf9317567b7``
+     - | Returns the value of ``ObjectId.self``:
```

Comment:
I am not sure about this one. `ObjectId.self` is undefined

<img width="550" alt="image" src="https://user-images.githubusercontent.com/761042/228443301-0961d83c-8b66-4bb2-87e2-563316faec40.png">

Maybe `Returns a formatted string`?